tests/unit/conftest.py

```diff
@@ -117,7 +121,9 @@ def met_pcc_pl(met_ecmwf_pl_path: str, override_cache: DiskCacheStore) -> MetDat
         paths=met_ecmwf_pl_path,
         cachestore=override_cache,
     )
-    return era5.open_metdataset()
+    met = era5.open_metdataset()
+    met.data = met.data.astype("float32")
```

Comment:
So I'm assuming you gave up on updating the nc fixtures?

Reply:
I'm fine with this as a way to fix the problem for now! If you want me to help re-run the fixtures, I'm happy to do this!

Reply:
I would still like to update the fixtures, but didn't want that to block releasing changes that Nick made. I'll start a conversation about that elsewhere.